doc/develop/modules.rst

```diff
@@ -569,6 +581,34 @@ Build files located in a ``MODULE_EXT_ROOT`` can be described as:
 This allows control of the build inclusion to be described externally to the
 Zephyr module.
 
+Vulnerability monitoring
+========================
+
+The module description file :file:`zephyr/module.yml` can be used for vulnerability monitoring
+and improve the generated SBOM output.
+
```

Comment:
Very good.

Maybe at this location specify the `version:` field should be used.
For example:
```suggestion

When a module monitors vulnerabilities then the module should also define the `version` field, :ref:`module_version`.
```

Reply:
Your comment makes me think this `version` field creates confusion. There was a misunderstanding from my part in the original issue.

The goal was to provide a version for the module and help complete the SBOM for the module (and not searching vulnerabilities from the forked repository for example). The `external-references` field must be used for that (and CPE and PURL specifications allow to include the version)

I think we should remove the `version` field from this PR and I will clarify this for a next PR. Are you okay with this ?

Reply:
It seems that there are some clarification (raised by your comment) to be done on this PR with the security group. In the meantime, I converted it into draft and I'll update it when I have all the information I need.

I'm sorry for this.

Reply:
if you believe it clarifies the usage and intentions of the field, then it's fine.

The clearer the intention of a PR, the better.